src/main/java/lotto/constant/WinningTier.java

```diff
@@ -0,0 +1,49 @@
+package lotto.constant;
+
+import java.util.List;
+
+public enum WinningTier {
+    FIRST(6, false, 2000000000),
+    SECOND(5, true, 3000000),
+    THIRD(5, false, 1500000),
+    FOURTH(4, false, 50000),
+    FIFTH(3, false, 5000),

```

Comment:
```suggestion
    FIRST(6, false, 2_000_000_000),
    SECOND(5, true, 3_000_000),
    THIRD(5, false, 1_500_000),
    FOURTH(4, false, 50_000),
    FIFTH(3, false, 5_000),

```
숫자의 가독성을 위해 _를 사용할 수 있습니다.

Reply:
생활꿀팁 감사합니다! 😆